useScroll/ScrollButtons.js

```diff
@@ -135,7 +134,7 @@ const useScrollButtons = (props) => {
 				if (focusableScrollButtons && !Spotlight.getPointerMode()) {
 					consumeEvent(ev);
 					Spotlight.setPointerMode(false);
-					Spotlight.focus(ReactDOM.findDOMNode(oppositeButton.ref)); // eslint-disable-line react/no-find-dom-node
+					Spotlight.focus(oppositeButton.ref);
```

Comment:
@daniel-stoian-lgp, It seems that ScrollButton ref does not work as we expected. 
In agate sampler, when focusableScrollbar set to true and testing with PageUp/Down button does not work properly. Could youe please check it?

Reply:
i updated the logic for ScrollButton.js
PAgeUp and PAgeDown between scrollbuttons seems to work now

Reply:
It seems that scrollbuttons work properly. Thank you.